showcase/src/app/news/news-item-component/news-item.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, Inject, Input, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, /*StarkErrorImpl, */ StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "news-item", // <news-item></news-item>
+	styleUrls: ["./_news-item.component.scss"],
+	templateUrl: "./news-item.component.html"
+})
+export class NewsItemComponent implements OnInit {
+	@Input() release: string;
+
+	@Input() newsDate: string;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public loggingService: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		this.loggingService.debug("hello from `News` component");
+
+		if (typeof this.release === "undefined" || this.release === "") {
+			console.log(this.release);
+			throw new Error("please provide the stark release");
+		}
+
+		if (typeof this.newsDate === "undefined" || this.newsDate === "") {
+			throw new Error("please provide a date for the news");
```

Comment:
Same remark:  include the name of the component at the beginning of any error message so it is easier to identify the source of the error